### PR TITLE
Rename LNbits primary key column from 'int' to 'id'

### PR DESCRIPTION
**Pull request by ekzyis**
## Description

This didn't cause any bugs because we use the id of the `Wallet` table.

## Checklist

**Are your changes backwards compatible? Please answer below:**

Yes, this shouldn't cause any downtime

**Did you QA this? Could we deploy this straight to production? Please answer below:**

Yes

**For frontend changes: Tested on mobile? Please answer below:**

**Did you introduce any new environment variables? If so, call them out explicitly here:**

No

### CODE REVIEW

File: prisma/migrations/20240821072645_rename_lnbits_pk/migration.sql [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+*/
+-- AlterTable
+ALTER TABLE "WalletLNbits" RENAME COLUMN "int" TO "id";
+
+UPDATE "Wallet"
+SET wallet = to_jsonb(NEW)
+WHERE id IN (
+  SELECT "walletId" FROM "WalletLNbits"
+)
```
Comment (huumn) [inline prisma/migrations/20240821072645_rename_lnbits_pk/migration.sql:15]: Is this valid sql? What does NEW refer to in an update statement?

Comment (huumn) [reply prisma/migrations/20240821072645_rename_lnbits_pk/migration.sql]: It wasn't. I fixed it.

**Comment by huumn** [comment]
> Did you QA this? Could we deploy this straight to production? Please answer below:
>
> Yes

👀 It's important we're able to trust this answer.

---------------

I've thought about changing this to "On a scale of 1-10 how well have your QA'd this change and any features it might affect? How did you QA this?"

File: prisma/migrations/20240821072645_rename_lnbits_pk/migration.sql (hunk 1 above)
Comment (ekzyis) [reply prisma/migrations/20240821072645_rename_lnbits_pk/migration.sql]: Whoops, I remember I ran an update query manually but must have forgotten to check what I committed

**Comment by ekzyis** [comment]
> I've thought about changing this to "On a scale of 1-10 how well have you QA'd this change and any features it might affect? How did you QA this?"

Good idea!

**Pull request merged**